packages/astro/src/core/session.ts

```diff
@@ -363,21 +363,30 @@ export class AstroSession<TDriver extends SessionDriverName = any> {
 		}
 
 		let driver: ((config: SessionConfig<TDriver>['options']) => Driver) | null = null;
-		const entry =
-			builtinDrivers[this.#config.driver as keyof typeof builtinDrivers] || this.#config.driver;
-		try {
-			// Try to load the driver from the built-in unstorage drivers.
-			// Otherwise, assume it's a custom driver and load by name.
+		const isBuiltin = this.#config.driver in builtinDrivers;
+		// Try to load the driver from the built-in unstorage drivers.
+		// Otherwise, assume it's a custom driver and load by name.
+		const driverPackage: string = isBuiltin
+			? builtinDrivers[this.#config.driver as keyof typeof builtinDrivers]
+			: this.#config.driver;
 
-			driver = await import(entry).then((r) => r.default || r);
+		try {
+			// If driver is not a builtin, and in development and test, load the driver directly.
```

Comment:
```suggestion
			// If driver is not a builtin, or in development or test, load the driver directly.
```
It seems the comment doesn't match the condition below

Reply:
Yeah it's poorly worded. It meant "a and b and c are situations where this applies"